runtime/vm/CRIUHelpers.cpp

```diff
@@ -128,6 +128,19 @@ isCheckpointAllowed(J9VMThread *currentThread)
 	return result;
 }
 
+
+BOOLEAN
+disableCRIUSecProvider(J9VMThread *currentThread)
+{
+	BOOLEAN result = FALSE;
+
+	if (isCRIUSupportEnabled(currentThread)) {
+		result = J9_ARE_ALL_BITS_SET(currentThread->javaVM->checkpointState.flags, J9VM_CRIU_DISABLE_CRIU_SEC_PROVIDER);
+	}
```

Comment:
Suggest this should also answer `TRUE` when CRIU support is not enabled:
```c
	BOOLEAN result = TRUE;

	if (isCRIUSupportEnabled(currentThread)) {
		result = J9_ARE_NO_BITS_SET(currentThread->javaVM->checkpointState.flags, J9VM_CRIU_DISABLE_CRIU_SEC_PROVIDER);
	}
```

Reply:
Updated

Reply:
Updated